appveyor.yml

```diff
@@ -15,40 +15,21 @@ install:
   - cmd: 7z x SHFBInstaller.zip -y
   - ps: msiexec /i InstallResources\SandcastleHelpFileBuilder.msi /quiet /qn /norestart /log install.log
 
-
 before_build:
-  - dotnet restore dotNetRDF.sln
```

Comment:
Is this now handled by Cake?

Reply:
Indirectly. `dotnet build` actually restores packages automatically